### PR TITLE
New files are added for BDT

### PR DESCRIPTION
**Pull request by ArielRojas**
Adding news files for BDT

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
 		"roomId" : "{:roomId}",
 		"outOfOrderId" : "{:outOfOrderId}"
 	},
-	"tokenBasicPrefix" : "Basic ",
+	"basicPrefix" : "Basic ",
```
Comment (AlejandroJeanCarlo) [inline config/config.json]: why you change this? this will be affect the project

File: lib/roomManagerAPI.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ var config=require(GLOBAL.initialDirectory+'/config/config');
 var request=require('superagent');
 var authorization=config.tokenHeader;
 var tokenPrefix=config.tokenPrefix;
-var basicPrefix = config.tokenBasicPrefix;
+var basicPrefix = config.basicPrefix;
```
Comment (AlejandroJeanCarlo) [inline lib/roomManagerAPI.js]: same

File: config/config.json (hunk 1 above)
Comment (ArielRojas) [inline config/config.json]: It does not affect the project

File: lib/roomManagerAPI.js (hunk 2 above)
Comment (ArielRojas) [inline lib/roomManagerAPI.js]: It does not affect the project

File: config/config.json (hunk 1 above)
Comment (ArielRojas) [inline config/config.json]: is used directly in the roomManagerAPI.js file

**Comment by ArielRojas** [comment]
The files roomMangerAPI.js and config.json are as before

**Pull request merged**